kibana-reports/server/routes/utils/report_template.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>%REPORT NAME%</title>
```

Comment:
Let's either add a generic title here, or insert the report title, if we're able to get it
```suggestion
    <title>Kibana Reports</title>
```